app/plugins/adp-common/README.md

```diff
@@ -0,0 +1,5 @@
+# @internal/plugin-adp-common
+
+Welcome to the common package for the adp plugin!
```

Comment:
Change the Welcome sentence to "Common functionalities, types, and permissions for the ADP plugin, to be shared between adp and adp-backend."